frontend/src/old-pages/Clusters/Cost.tsx

```diff
@@ -0,0 +1,293 @@
+      setDateValue(Default);
+  },[])
+
+  return (
+    <Container
+      header={
+        <Header
+          variant="h2"
+          actions={
+            <SpaceBetween
+              direction="horizontal"
+              size="xs"
+            >
+              <DateRangePicker
+              onChange={({ detail }) => {fetchGraphData(detail.value); setDateValue(detail.value); }}
+                value={dateValue}
+                relativeOptions={[
+                  {
+                    key: "previous-30-days",
+                    amount: 30,
+                    unit: "day",
+                    type: "relative"
+                  },
+                  {
+                    key: "previous-14-days",
+                    amount: 14,
+                    unit: "day",
+                    type: "relative"
+                  },
+                  {
+                    key: "previous-7-days",
+                    amount: 7,
+                    unit: "day",
+                    type: "relative"
+                  },
+                  {
+                    key: "previous-1-day",
+                    amount: 1,
+                    unit: "day",
+                    type: "relative"
+                  }
+                ]}
+                i18nStrings={{
+                  todayAriaLabel: "Today",
+                  nextMonthAriaLabel: "Next month",
+                  previousMonthAriaLabel: "Previous month",
+                  customRelativeRangeDurationLabel: "Duration",
+                  customRelativeRangeDurationPlaceholder:
+                    "Enter duration",
+                  customRelativeRangeOptionLabel: "Custom range",
+                  customRelativeRangeOptionDescription:
+                    "Set a custom range of days in the past. Use Days as Unit of Time.",
+                  customRelativeRangeUnitLabel: "Unit of time",
+                  formatRelativeRange: e => {
+                    const t =
+                      1 === e.amount ? e.unit : `${e.unit}s`;
+                    return `Last ${e.amount} ${t}`;
+                  },
+                  formatUnit: (e, t) => (1 === t ? e : `${e}s`),
+                  dateTimeConstraintText:
+                    "Select any Range of days. Use 24 hour format.",
+                  relativeModeTitle: "Relative range",
+                  absoluteModeTitle: "Absolute range",
+                  relativeRangeSelectionHeading: "Choose a range",
+                  startDateLabel: "Start date",
+                  endDateLabel: "End date",
+                  startTimeLabel: "Start time",
+                  endTimeLabel: "End time",
+                  clearButtonLabel: "Clear and dismiss",
+                  cancelButtonLabel: "Cancel",
+                  applyButtonLabel: "Apply"
+                }}
+                placeholder="Filter by a date range" />
+        <Button disabled={tag_active} onClick={activate_tags} variant="primary">Activate Tags</Button> 
```

Comment:
This tab is cluster specific, however the activation of tags seems to be account specific? If so then this probably belongs somewhere else (a new settings page?)

Reply:
This is outside of the scope of the project. Since activating tags is a one-time thing and there's no cost associated with it, it's a safe operation for any user (not just cluster admins) to do. For organizational purposes makes more sense on a settings page but again a bit outside of scope.